components/feature/downloads/src/test/java/mozilla/components/feature/downloads/DownloadsFeatureTest.kt

```diff
@@ -267,6 +268,36 @@ class DownloadsFeatureTest {
         verify(dialogFragment, never()).showNow(any(), any())
     }
 
+    @Test
+    fun `WHEN calling onCancelDownload THEN closeDownloadResponse and consumeDownload should be called`() {
```

Comment:
nit: the test names should describe the behaviour not the chain of methods that are called. If we change these method names later, all the tests will have wrong names :).

Better: WHEN a download is cancelled THEN the response is closed ....etc.

Reply:
👍🏽